modules/reindex/src/main/java/org/elasticsearch/reindex/TransportUpdateByQueryAction.java

```diff
@@ -52,18 +54,21 @@ public TransportUpdateByQueryAction(
         Client client,
         TransportService transportService,
         ScriptService scriptService,
-        ClusterService clusterService
+        ClusterService clusterService,
+        UpdateByQueryMetrics updateByQueryMetrics
```

Comment:
maybe we should annotate this variable as `@Nullable` too?

Reply:
Yes, will do.